baseapp-chats/baseapp_chats/base.py

```diff
@@ -53,7 +55,7 @@ class Verbs(models.IntegerChoices):
     def description(self):
         return self.label
 
-    content = models.TextField(null=True, blank=True)
+    content = RichTextField(blank=True, null=True)
```

Comment:
RichTextField worries me a lot, it means we are dealing with raw HTML and that can come with major security implications, I would look into using a more simple formatting language like markdown for example, its a big reason to why github comments uses it for example